test/TesterInternal/StreamingTests/SQSStreamTests.cs

```diff
@@ -8,62 +8,59 @@
 using UnitTests.StorageTests.AWSUtils;
 using UnitTests.StreamingTests;
 using Xunit;
+using Orleans.Runtime.Configuration;
 
 namespace UnitTests.Streaming
 {
     [TestCategory("AWS"), TestCategory("SQS")]
     public class SQSStreamTests : HostedTestClusterPerTest
     {
-        internal static readonly FileInfo SiloConfigFile = new FileInfo("Config_SQSStreamProviders.xml");
-        internal static readonly FileInfo ClientConfigFile = new FileInfo("ClientConfig_SQSStreamProviders.xml");
         public static readonly string SQS_STREAM_PROVIDER_NAME = "SQSProvider";
 
-        private static readonly TestingSiloOptions sqsSiloOptions = new TestingSiloOptions
-        {
-            SiloConfigFile = SiloConfigFile,
-        };
-        private static readonly TestingClientOptions sqsClientOptions = new TestingClientOptions
-        {
-            ClientConfigFile = ClientConfigFile
-        };
-
         private SingleStreamTestRunner runner;
 
-        public override TestingSiloHost CreateSiloHost()
+        public override TestCluster CreateTestCluster()
         {
+            var options = new TestClusterOptions(initialSilosCount: 4);
```

Comment:
by looking at the diff, it doesn't look as it was starting 4 silos, it was probably just starting primary and secondary. Haven't looked at the XML config, but still if there are more nodes defined there, it doesn't necessarily mean that those nodes will be started, it's just that the config overrides are prepared for it.